test/format/config.c

```diff
@@ -989,18 +996,32 @@ config_mirrors(void)
     explicit_mirror = config_explicit(NULL, "runs.mirror");
     if (!explicit_mirror && mmrand(NULL, 1, 10) < 9)
         return;
-    config_off_all("runs.mirror");
 
     /*
      * We can't mirror if we don't have enough tables. A FLCS table can be a mirror, but it can't be
      * the source of the bulk-load mirror records. Find the first table we can use as a base.
      */
     for (i = 1; i <= ntables; ++i)
-        if (tables[i]->type != FIX)
+        if (tables[i]->type != FIX && !NT_EXPLICIT_OFF(tables[i], RUNS_MIRROR))
             break;
-    if (ntables < 2 || i > ntables) {
-        if (explicit_mirror)
-            WARN("%s", "table selection didn't support mirroring, turning off mirroring");
+
+    if (i > ntables && explicit_mirror) {
+        WARN("%s", "table selection didn't support mirroring, turning off mirroring");
+        return;
+    }
+
+    /*
+     * We also can't mirror if we don't have enough tables that have allowed mirroring. It's
+     * possible for a table to explicitly set runs.mirror=0, so check how many tables have done
```

Comment:
I do think I'd prefer tableN.runs.mirror=0 because that is what we see in the CONFIG so it is clearer to me at least.

Reply:
No problem, I'll update it.

Reply:
Fixed in 74f738e